<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Receiver</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="4">
        <widget class="QProgressBar" name="mpcReceiveProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="mpcSelectDirBtn">
         <property name="text">
          <string>Set Directory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="mpcStatusLabel">
         <property name="text">
          <string>Status?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="mpcListenBtn">
         <property name="text">
          <string>Listen</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="mpcPortSb">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>12345</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLabel" name="mpcReceivingDirLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QPlainTextEdit" name="mpcIpInfo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sender</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QSpinBox" name="mpcServerPortSb">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="mpcConnectBtn">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="mpcSendBtn">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QProgressBar" name="mpcProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="mpcServerIpLe"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="mpcFilesToSendBtn">
         <property name="text">
          <string>Select Files To Send</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="mpcConnectionStatusLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
